tests/41end-to-end-keys/07-backup.pl

```diff
@@ -347,6 +349,19 @@
       });
    };
 
+test "Can create more than 10 backup versions",
+   requires => [ $fixture ],
+
+   bug => 'synapse/4169',
```

Comment:
I don't think this does what you want. It marks the test as an "expected failure".

Reply:
I assume it only does this while the bug is still open though?

Reply:
You'd be wrong :) The string isn't interpreted at all, and certainly doesn't go and check the status of any linked issue (nor is there any consistency in the format of the string)

Reply:
looks like you're right though